Orm/Xtensive.Orm.Tests/Issues/IssueJira0614_TypeMappingCachingInMaterializationContext.cs

```diff
@@ -375,45 +379,43 @@ public void Test02()
 
       var multinodeDomainConfiguration = CreateConfiguration(typeof(Target.TimesheetCode), DomainUpgradeMode.PerformSafely, "Model2");
       var nodeConfiguration = CreateNodeConfiguration(Node1Name, "Model2", "Model1", DomainUpgradeMode.PerformSafely);
+
       using (var domain = BuildDomain(multinodeDomainConfiguration, nodeConfiguration)) {
-        using (var session = domain.OpenSession()) {
-          session.SelectStorageNode(WellKnown.DefaultNodeId);
+        var selectedNode = domain.SelectStorageNode(WellKnown.DefaultNodeId);
```

Comment:
Let's instead consider direct usage of `StorageNodeManager` as in the sample below.
```csharp
var storageNode = domain.StorageNodeManager.GetNode(WellKnown.DefaultNodeId);
using (var session = storageNode.OpenSession()) {
  ...
}
```

Reply:
I thought about it but ```StorageNode``` has no relation to Domain now. ```StorageNode``` is stand-alone item to manage by ```StorageNodeManager```. I didn't want to change it. Plus in my opinion ```Domain``` instance is what people have in, for example, service containers, I would. People are used to it.

Another thing is that ```StorageNode``` is mostly used by us. Not so many people use metadata it stores within. We can transform StorageNode to something people work with on regular basis, like opening sessions, but in this case the workflow should be easy-to-use and clear-to-understand. Opening a session to a StorageNode by the StorageNode looks kind of logical for multi-node ```Domain``` at first glance. Any cons of that?

Reply:
I don't see any cons.